config/olm/kubernetes/0.2.1/manifests/dynatrace-operator.v0.2.1.clusterserviceversion.yaml

```diff
@@ -40,7 +40,7 @@ metadata:
       ]
     capabilities: Deep Insights
     categories: Monitoring,Logging & Tracing
-    containerImage: docker.io/dynatrace/dynatrace-operator
+    containerImage: docker.io/dynatrace/dynatrace-operator:v0.2.1
     operators.operatorframework.io/builder: operator-sdk-v1.5.0
```

Comment:
the latest operator sdk is 1.13.0, please double check that this value is correct